harmoni_actuators/harmoni_tts/aws_tts_service.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python3
+
+# Common Imports
+import rospy
+import roslib
+
+from harmoni_common_lib.constants import State
+from harmoni_common_lib.service_server import HarmoniServiceServer
+from harmoni_common_lib.service_manager import HarmoniServiceManager
+import harmoni_common_lib.helper_functions as hf
+
+# Specific Imports
+from harmoni_common_lib.constants import ActuatorNameSpace
+from botocore.exceptions import BotoCoreError, ClientError
+from contextlib import closing
+import soundfile as sf
+import numpy as np
+import boto3
+import re
+import json
+import ast
+import sys
+
+
+class AWSTtsService(HarmoniServiceManager):
```

Comment:
I would change the name to `AWS_TTS_Service` since it's all acronyms.

Reply:
Thank you for the suggestion, I changed the name for the file in /harmoni_tts/nodes